storage/zswap.txt

```diff
@@ -0,0 +1,92 @@
+|           Compressed cache for swap pages default compressor (xxx)           |
+|           Compressed cache for swap pages default allocator (xxx)            |
+|       <*> Enable the compression cache for swap pages by default             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Either set CONFIG_ZSWAP_DEFAULT_ON=y in the kernel config or add zswap_enabled=1
+to your kernel command line (via the bootloader or the built-in command line) to
+have zswap enabled at boot-time. To enable zswap at runtime,
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ echo 1 > /sys/module/zswap/parameters/enabled                            |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Zswap has many options that can also be configured at runtime, including which
+compressor is in use. To see them all, 
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ grep -R . /sys/module/zswap/parameters                                   |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+For information on setting up a swapfile or swap partition, see @/storage/disks.
+
+
+zbud versus z3fold versus zmalloc
+________________________________________________________________________________
+
+There are three different allocators to choose from for compressed pages:
+
++-------------+----------------------------------------------------------------+
+| Allocator   | Description                                                    |
+|-------------+----------------------------------------------------------------|
+|             |                                                                |
+| zbud        | Uses a 2:1 compressed:uncompressed page allocation (legacy)    |
+| z3fold      | Uses a 3:1 ratio                                               |
+| zsmalloc    | Designed for zram - better under low memory conditions         |
+|             |                                                                |
++-------------+----------------------------------------------------------------+
+
+In general, z3fold should be preferred to zbud; the latter is supported solely
+for compatibility purposes. z3fold provides a better compression ratio and
+should be preferred when possible. 
+
+zsmalloc has a very different page allocation method than either zbud or z3fold,
+and provides for greater storage density. However, zsmalloc does not implement
```

Comment:
`for` -> `far` ?

Reply:
This may be correct already though... both make sense. :P

Reply:
They do both make sense; if 'provides for' is too pretentious sounding I can change it XD